pkgs/development/python-modules/ripe-atlas-sagan/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, python-dateutil
+, pytz
+, cryptography
+, nose
+, fetchFromGitHub }:
+buildPythonPackage rec {
+  pname = "ripe-atlas-sagan";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "RIPE-NCC";
+    repo = "ripe-atlas-sagan";
+    rev = "v${version}";
+    sha256 = "sha256-xIBIKsQvDmVBa/C8/7Wr3WKeepHaGhoXlgatXSUtWLA=";
+  };
+
+  propagatedBuildInputs = [
+    python-dateutil
+    pytz
+    cryptography
+  ];
+
+  checkInputs = [
+    nose
+  ];
```

Please add `pythonImportsCheck`. Something like this should check if the module can be imported.

```suggestion
  ];

  pythonImportsCheck = [
    "ripe.atlas.sagan"
  ];
```